# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1545496073 -3600
#      Sat Dec 22 17:27:53 2018 +0100
# Branch stable
# Node ID 9644c0b7efd8c39f9149fd3f3763cb35c8273c24
# Parent  0d755097d73f55cdbf35c4940ace7036936b72d1
doc: CITATION update year and version for final release.

* CITATION: update year to 2019 (scheduled for end of January 2019) and version
to final release 5.1.0 (see etc/HACKING.md).

diff --git a/CITATION b/CITATION
--- a/CITATION
+++ b/CITATION
@@ -1,19 +1,19 @@
 To cite GNU Octave in publications use:
 
-  John W. Eaton, David Bateman, Søren Hauberg, Rik Wehbring (2018).
-  GNU Octave version 4.4.1 manual: a high-level interactive language for
+  John W. Eaton, David Bateman, Søren Hauberg, Rik Wehbring (2019).
+  GNU Octave version 5.1.0 manual: a high-level interactive language for
   numerical computations.
-  URL https://www.gnu.org/software/octave/doc/v4.4.1/
+  URL https://www.gnu.org/software/octave/doc/v5.1.0/
 
 A BibTeX entry for LaTeX users is:
 
   @manual{,
-    title     = {{GNU Octave} version 4.4.0 manual: a high-level interactive language for numerical computations},
+    title     = {{GNU Octave} version 5.1.0 manual: a high-level interactive language for numerical computations},
     author    = {John W. Eaton and David Bateman and S{\o}ren Hauberg and Rik Wehbring},
-    year      = {2018},
-    url       = {https://www.gnu.org/software/octave/doc/v4.4.1/},
+    year      = {2019},
+    url       = {https://www.gnu.org/software/octave/doc/v5.1.0/},
   }
 
 We have invested a lot of time and effort in creating GNU Octave, please
 cite it when using it.  See also `citation pkgname' for citing Octave
 packages.
